snippets/scripts/jira-components-to-csv/jira_components_to_csv.py

```diff
@@ -0,0 +1,42 @@
+# Prereqs: pip/pip3 install -r requirements.txt
+
+import requests
+import pandas as pd
+
+# Replace with your email, API token, site URL, and Jira project keys
+email = "your-email"
+api_token = "your-api-token"
+site_url = "your-site-url" # desired format: my-cool-site.atlassian.net
+project_keys = ['your-project-key', 'another-project-key'] # List of your three-letter Jira project keys
```

Comment:
May be a bit misleading, as project keys are not necessarily three letters 😅 

```suggestion
project_keys = ['your-project-key', 'another-project-key'] # List of Jira project keys or ids to extract
```